### PR TITLE
embed diagrams in readme

### PR DESCRIPTION
**Pull request by WillBanks1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ For a visual representation of the project architecture and details on the desig
 
 ### Architecture Diagram
 
-![Architecture Diagram](TO BE FILLED)
+![Architecture Diagram](https://github.com/fm1psy/c11-poseidon-earthquake-monitoring/blob/main/diagrams/earthquake_architecture.png)
```
Comment (joe1606) [inline README.md:21]: Not sure if this works? I tried opening a preview and no picture showed up

File: README.md [hunk 2]
```diff
@@ -28,9 +28,9 @@ For a visual representation of the database schema and details on the design dec
 
 ### Entity-Relationship Diagram (ERD)
 
-<img src="./ERD.png" alt="ERD" width="800"/>
+The provided Entity-Relationship Diagram (ERD) illustrates a database schema designed for tracking earthquake data. The schema follows the principles of Third Normal Form (3NF). There is a main earthquake table that captures data specific to each earthquake event. Additionally, lookup tables are included for categories such as status, alerts and networks.
 
-The provided Entity-Relationship Diagram (ERD) illustrates a database schema designed for tracking earthquake data. The schema follows the principles of Third Normal Form (3NF). There is a main earthquake table that captures data specific to each earthquake event. Additionally, lookup tables are included for categories such as status, alerts, networks, types, and magtypes.
+![ERD Diagram](https://github.com/fm1psy/c11-poseidon-earthquake-monitoring/blob/main/diagrams/erd_diagram.png)
```
Comment (joe1606) [inline README.md:33]: same as above

**Review by joe1606 — COMMENTED** [review]
Just want to double check before approving

**Comment by joe1606** [comment]
closes #24

**Review by fm1psy — APPROVED** [review]
Looks good

**Pull request merged**